service/framework.go

```diff
@@ -265,7 +265,9 @@ func newCustomObjectFramework(config Config) (*framework.Framework, error) {
 			if err != nil {
 				return nil, microerror.Mask(err)
 			}
+
 			container.KeyVals["cluster"] = key.ClusterID(customObject)
+			container.KeyVals["framework"] = "customobject"
```

Comment:
I am not sure if the wording is good. Feedback appreciated. I just want to have something I can grep for to get serialized logs from the same reconciliation loop. I already called it framework, custom object and pod in the code. I don't mind.

Reply:
reconciliation?

Reply:
What do you mean? Should the framework here be called `reconciliation`? The other framework reconciles as well. I thought the framework should be identified for what it is reconciling or watching.

Reply:
`container.KeyVals["reconciliation"] = "customobject"`

Reply:
Feel free to provide a PR and name it with what you think makes sense.

Reply:
For the record it is done in https://github.com/giantswarm/kvm-operator/pull/266